Sources/Moya/Plugins/NetworkLoggerOutput.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NetworkLoggerOutput.swift
+//  Moya
+//
+//  Created by Alan Yeo on 24/4/18.
+//
+
+import Foundation
+
+public protocol NetworkLoggerOutput {
```

Comment:
Nit: Think we need some work on naming here. 
Maybe `NetworkLoggerOutputFormatter`? Need advice from another contributor